tests/dags/common/loader/test_reporting.py

```diff
@@ -0,0 +1,26 @@
+from unittest import mock
+
+import pytest
+from common.loader.reporting import report_completion
+
+
+@pytest.fixture(autouse=True)
+def send_message_mock() -> mock.MagicMock:
+    with mock.patch("common.slack.send_message") as SendMessageMock:
+        yield SendMessageMock.return_value
```

Comment:
Should this just be `yield SendMessageMock`? I think we want to check that the function itself was called, not that the function's return value was. I might be interpreting this wrong

Reply:
No I think you're right! Fixed, thanks for the catch 👍